docs/java/tips-and-tricks/common-problems.md

```diff
@@ -0,0 +1,59 @@
+---
+id: sdk-java-common-problems
+title: Some common problems and how to address them
```

Comment:
```suggestion
title: How to fix common issues?
```

Reply:
I would not write "how to fix" because the article is really only meant to be a collection pointing to further resources. From a "how to fix" I would expect something like "Enable debug logs, try this solution, if that doesn't work try this, etc.".